includes/data-port/class-sensei-import-job.php

```diff
@@ -277,4 +277,43 @@ public static function get_default_results() {
 
 		return $results;
 	}
+
+	/**
+	 * Returns the post id for an import id or check if the post exists.
+	 *
+	 * @param string $post_type  The post type.
+	 * @param string $import_id  The import id.
+	 *
+	 * @return int|null The post id if the post exists, null otherwise.
+	 */
+	public function translate_import_id( $post_type, $import_id ) {
```

Comment:
This method was moved from `includes/data-port/models/class-sensei-import-model.php`